src/pages/CreateKas/index.tsx

```diff
@@ -0,0 +1,11 @@
+import { CreateKasProvider } from "@pages/CreateKas/context";
+import CreateKasLayout from "./layout";
+const index = () => {
```

Comment:
```suggestion
import CreateKasLayout from "./layout";

const index = () => {
```
kasih newline antar import dan isi